test/src/test/java/org/corfudb/integration/AbstractIT.java

```diff
@@ -305,8 +306,8 @@ public static CorfuRuntime createRuntimeWithCache() {
         return rt;
     }
 
-    public static Map<String, Integer> createMap(CorfuRuntime rt, String streamName) {
-        Map<String, Integer> map = rt.getObjectsView()
+    public static StreamingMap<String, Integer> createMap(CorfuRuntime rt, String streamName) {
```

Comment:
Why is this needed?

Reply:
`Checkpointer` needs to consume an interface that exposes the streaming API, so `Map` will have to be replaced with something else. Most likely `ISMRMap`.